tests/unit/serializers/tumblr-post-test.js

```diff
@@ -6,10 +7,25 @@ moduleForModel('tumblr-post', 'Unit | Serializer | tumblr-post', {
 });
 
 // Replace this with your real tests.
-test('it serializes records', function(assert) {
-  var record = this.subject();
+test('it serializes records', function (assert) {
+  const record = this.subject();
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).